db/migrate/20200219125312_add_thin_and_format_to_disk.rb

```diff
@@ -0,0 +1,6 @@
+class AddThinAndFormatToDisk < ActiveRecord::Migration[5.1]
+  def change
+    add_column :disk, :thin, :boolean
```

Comment:
Ah this needs to be `:disks` not `:disk`

Reply:
done